code-reviews/example3/adventure_game.py

```diff
@@ -0,0 +1,79 @@
+# An interactive adventure game!
+import random
+
+def print_welcome():
+    print("Welcome to the adventure game! 🛡")
+    print("Maintain your health while finding food 🥕")
+    print("But watch out for the wild animals! 🦊")
```

Comment:
Wht about a lion instead of the fox,won't it look cooler?

Reply:
Lion would would be cooler here :lion:

Reply:
Yeah man